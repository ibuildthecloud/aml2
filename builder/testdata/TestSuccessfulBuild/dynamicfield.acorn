a: "x"
(a): 1
