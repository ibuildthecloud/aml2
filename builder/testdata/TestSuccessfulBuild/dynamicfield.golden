&errors.joinError{errs: []error{
	&builder.ErrUnknownError{Node: &ast.ParenExpr{
		Lparen: token.Pos{
			file: &token.File{
				name: "dynamicfield.acorn",
				base: token.index(1),
				size: token.index(14),
				lines: []token.index{
					token.index(0),
					token.index(7),
				},
			},
			offset: 132,
		},
		X: &ast.Ident{
			NamePos: token.Pos{
				file: &token.File{
					name: "dynamicfield.acorn",
					base: token.index(1),
					size: token.index(14),
					lines: []token.index{
						token.index(0),
						token.index(7),
					},
				},
				offset: 146,
			},
			Name: "a",
		},
		Rparen: token.Pos{
			file: &token.File{
				name: "dynamicfield.acorn",
				base: token.index(1),
				size: token.index(14),
				lines: []token.index{
					token.index(0),
					token.index(7),
				},
			},
			offset: 162,
		},
	}},
}}
